packages/react-data-grid/src/RowComparer.js

```diff
@@ -32,6 +32,7 @@ export const shouldRowUpdate = (nextProps, currentProps) => {
     currentProps.isSelected !== nextProps.isSelected ||
     nextProps.height !== currentProps.height ||
     currentProps.isOver !== nextProps.isOver ||
+    currentProps.expandedRows !== nextProps.expandedRows ||
```

Comment:
Need two tests that the RowComparer returns the correct boolean value for expanded rows inputs